### PR TITLE
Embellish third-party section with other discovered usage

### PR DESCRIPTION
**Pull request by andrewpollock**
Worth periodically reviewing the results from
https://cs.github.com/?q=%22api.osv.dev%22 to see what else pops up.

Also ran through mdformat

Fixes: #464

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,16 +118,27 @@ git submodule update --init --recursive
...
 
-There are also community tools that use OSV. Note that these are community built tools and
-unsupported by the core OSV maintainers.
+There are also community tools that use OSV. Note that these are community built
+tools and unsupported by the core OSV maintainers.
 
-- [G-Rath/osv-detector](https://github.com/G-Rath/osv-detector): A scanner that uses the OSV database.
+-   [G-Rath/osv-detector](https://github.com/G-Rath/osv-detector): A scanner
+    that uses the OSV database.
+-   [Rust client library](https://github.com/gcmurphy/osv)
```
Comment (oliverchang) [inline README.md:131]: Also https://github.com/DependencyTrack/dependency-track

**Review by oliverchang — COMMENTED** [review]
thanks! Just one suggestion to add.

File: README.md (hunk 1 above)
Comment (andrewpollock) [reply README.md:131]: Thanks, I wonder how that one didn't turn up in my search?

File: README.md [hunk 2]
```diff
@@ -106,16 +118,28 @@ git submodule update --init --recursive
...
+-   [Golang support for the schema](https://pkg.go.dev/golang.org/x/vuln/osv)
+-   [.NET client library and support for the schema](https://github.com/JamieMagee/osv.net)
+-   [EZE-CLI: The one stop shop for security testing in modern development](https://github.com/RiverSafeUK/eze-cli)
+-   [OSS Review Toolkit](https://github.com/oss-review-toolkit/ort)
+-   [Skjold: Security audit python project dependencies against several security
+    advisory databases](https://github.com/twu/skjold)
+-   [Cortex XSOAR](https://github.com/demisto/content)
+-   [dependency track](https://github.com/DependencyTrack/dependency-track)
```
Comment (oliverchang) [inline README.md:139]: nit: Maybe stylise as "Dependency-Track" per the repo description and website?

Comment (andrewpollock) [reply README.md]: Done. I'd based it off the logo text.

**Review by oliverchang — APPROVED** [review]

**Pull request merged**